AdvancedReports.php

```diff
@@ -594,6 +713,22 @@ function outputViewReportHeader( $reportLabel, $reportType )
 
 		}
 
+		// If applicable for the report type, show a link to reset the report state.
+		if ( $canReset )
+		{
+
+?>
+ <span id="mod-advrep-resetstate" style="display:none">
```

Comment:
Should this be hidden?

Reply:
Yes, it starts hidden and JavaScript will unhide it as required.